src/entity/active_model.rs

```diff
@@ -843,4 +908,175 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    #[cfg(feature = "with-json")]
+    #[should_panic(
+        expected = r#"called `Result::unwrap()` on an `Err` value: Json("missing field `id`")"#
```

Comment:
I don't understand why this should panic where `test_active_model_set_from_json_1` does not

Reply:
`test_active_model_set_from_json_1` will fail because `cake::Model` doesn't specify `#[serde(skip_deserializing)]`.

https://github.com/SeaQL/sea-orm/blob/b32e0227168b2455aa78ab172d2c93fb9fa41f3e/src/tests_cfg/cake.rs#L7-L15

While `fruit::Model` specify it and prevent the error in `test_active_model_set_from_json_2`.

https://github.com/SeaQL/sea-orm/blob/b32e0227168b2455aa78ab172d2c93fb9fa41f3e/src/tests_cfg/fruit.rs#L7-L16

Reply:
got it